tests/lit-tests/2650.ispc

```diff
@@ -0,0 +1,12 @@
+// RUN: %{ispc} --target=host --nostdlib --nowrap -h %t.h %s 2>&1
```

Comment:
Why not using pipe here? I.e. `... 2>&1 | FileCheck ...`

Reply:
Ah, I missed that output is not streamed as well. I suggest doing `%{ispc} --target=host --nostdlib --nowrap %s -h - 2>&1 | FileCheck %s`.

This would also check that `-` properly works for header output. Not a big deal, but would not harm.

Reply:
@dbabokin the output appears to be used _twice_, once in `FileCheck` and once in `cc`, so that might not work?

Reply:
> Ah, I missed that output is not streamed as well. I suggest doing `%{ispc} --target=host --nostdlib --nowrap %s -h - 2>&1 | FileCheck %s`.

It doesn't work as expected. It just creates the header file names `-`.

> This would also check that `-` properly works for header output. Not a big deal, but would not harm.

It doesn't look like working at the moment.